docs/installation/local.md

```diff
@@ -4,4 +4,46 @@ sidebar_position: 1
 
 # Local Installation
 
-For development purposes, you can install Capact locally. To learn how to do that, see the [Development cluster](../development/development-guide.md#development-cluster) section in the "Development Guide" document.
+Use Capact CLI to create local Kubernetes cluster and install Capact on it.
+
+## Prerequisites
+- [Docker](https://www.docker.com/) installed
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed
+- Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide
+
+## Create local Kubernetes cluster
+
+To create local [kind](https://kind.sigs.k8s.io/) cluster, run the following command:
+
+```bash
+capact environment create kind
```

Comment:
```suggestion
capact environment create kind --wait 5m
```
Previously we had 5min, IMO it's max that it may take. Normally, it's under 1min.